src/bundleList.js

```diff
@@ -127,6 +131,13 @@ function initBundleBuckets(roles) {
 function sortBundleByBuckets(roles, bundle, bundleBuckets) {
   roles.forEach((role) => {
     if (bundle.indexOf('-' + role + '-') !== -1) {
+
+      // skip invalid bundle names
+      if( !validBundleRegex.test( bundle ) ){
+        console.error( 'skip bundle', bundle );
```

Comment:
I'm thinking about how this error would be interpreted by users running this download and I feel like it could be confusing. Should we remove or reword it? In any case it is not an error if we are intentionally skipping that download. Maybe it's better as a `.info` log entry?